docs/emcc.txt

```diff
@@ -322,9 +322,8 @@ Options that are modified or new in *emcc* are listed below:
 
    Note:
 
-     Embedding files is much less efficient than preloading them. You
-     should only use it for small files, in small numbers. Instead use
-     "--preload-file", which emits efficient binary data.
+     Embedding files is genearlly more efficient than preloading them
+     can avoid copying/duplicating the file data.
```

Comment:
```suggestion
     as it avoids copying/duplicating the file data.
```

Reply:
Can you expand this a bit? Copying/duplicating the file data where? Does it both duplicate (at compile time) and copy (at runtime)?

Reply:
With preloading the file data is download by JS and copied in the linear memory, so it is both copied and (until GC runs anyway) duplicated in memory (1 copy in native, 1 in JS).

Reply:
Happy to rephrase but I think maybe keeping it simple is good.  Should I just say "copying"?

Reply:
I re-phased it.